k8s/esp_echo_gke_ingress.yaml

```diff
@@ -40,7 +40,7 @@ spec:
     spec:
       containers:
       - name: esp
-        image: b.gcr.io/endpoints/endpoints-runtime:1.0
+        image: b.gcr.io/endpoints/endpoints-runtime:latest
```

Comment:
do we have "latest" tag now?
and please also document this is just for samples, which versions should be used for production. We probably should have a page about released versions for production use